src/browser/utils.ts

```diff
@@ -74,20 +74,41 @@ function clamp(num: number, min: number = 0, max: number = Number.MAX_SAFE_INTEG
 }
 
 /*
-  Extends raw native window info.
+  Parses raw native window info object and returns a "clean" info object.
 */
-export function extendNativeWindowInfo(rawNativeWindowInfo: Shapes.RawNativeWindowInfo): Shapes.NativeWindowInfo {
+export function parseRawNativeWindowInfo(
```

Comment:
alternative names to consider - getNativeWinInfoFromRawData / extractNativeWinInfoFromRawData

Reply:
Not sure..., the thing is, props will be modified, removed or deleted. `get` is kinda there but not really. `extract` I don't think is a better word.